include/roboteam_ai/world/Field.h

```diff
@@ -33,6 +33,24 @@ struct FieldArc {
  * - The location and sizes of all arcs on the field. <br>
  * - Important and frequently used locations of the field, e.g. positions around our and the opponents goal.
  *
+ * The encoding of the field looks like:
+ *
+ *                          (upperFieldLine)
+ *              (-x,y)_________________________ (x,y)
+ *                 |                              |
+ *                 |                              |
+ * (your side)     |             (0,0)            | (their side)
+ * (leftFieldLine) |                              | (rightFieldLine)
+ *                 |                              |
+ *              (-x, -y)_______________________(x, -y)
+ *                          (bottomFieldLine)
+ *
+ * Thus the left side of the field always corresponds to our side of the field and the right side of the field always corresponds to the opponents side of the field. Moreover
```

Comment:
```suggestion
 * Thus the left side of the field always corresponds to our side of the field. Moreover
```

Reply:
I do not agree, that part of the explanation is important...

Reply:
[https://me.me/i/there-are-two-types-of-people-in-this-world-1-15505643](A scientific paper about why you are wrong)